recipes/roadies/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "0.1.4" %}
+
+package:
+  name: roadies
+  version: {{ version }}
+
+source:
+  url: https://github.com/TurakhiaLab/ROADIES/archive/v{{ version }}.tar.gz
+  sha256: e3b4b89f247824c4b61e790f094320624bde47e295e2158a9ad83c05dfa83d54
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("roadies", max_pin=None) }}
```

Comment:
Are you sure about 'None' here?
See the Pull request template for its meaning.

Reply:
Thanks for bringing this up. We are working on updating our repo to support semantic versioning.